Userland/Libraries/LibUnicode/CodeGenerators/GenerateUnicodeData.cpp

```diff
@@ -404,14 +404,15 @@ constexpr @name@ operator|(@name@ value1, @name@ value2)
 
 #include <AK/Optional.h>
 #include <AK/Types.h>
+#include <LibUnicode/Forward.h>
 
 namespace Unicode {
 )~~~");
 
     generate_enum("Locale"sv, "None"sv, move(unicode_data.locales));
     generate_enum("Condition"sv, "None"sv, move(unicode_data.conditions));
     generate_enum("GeneralCategory"sv, {}, move(unicode_data.general_categories));
-    generate_enum("Property"sv, "Assigned"sv, unicode_data.prop_list.keys(), move(unicode_data.prop_aliases), true);
+    generate_enum("Property"sv, "Assigned"sv, unicode_data.prop_list.keys(), unicode_data.prop_aliases, true);
```

Comment:
This is `move()`d two commits earlier (`LibUnicode: Parse and generate PropertyAliases`), if that's incorrect and reverted here, can we avoid making the change in the first place?

Reply:
Yeah sure, I think so. Wasn't really incorrect per-se in the first commit - at that point I moved it because we were done with it. Then in this commit, I needed it kept alive to generate `property_from_string`.

Reply:
alright, that's ok then :+1: